fili-core/src/test/groovy/com/yahoo/bard/webservice/data/PartialDataHandlerSpec.groovy

```diff
@@ -99,7 +99,7 @@ class PartialDataHandlerSpec extends Specification {
          * starts inside the dim1 hole and goes to the end of the period.
          */
         SegmentMetadata segments = new SegmentMetadata(
-                [("user_device_type"): buildIntervals(["2014-07-01/2014-07-09","2014-07-11/2014-07-29"]) as LinkedHashSet,
+                [("userDeviceType"): buildIntervals(["2014-07-01/2014-07-09","2014-07-11/2014-07-29"]) as LinkedHashSet,
```

Comment:
We changed internal storage of `Column` to use `apiName' instead of `physicalName`